packages/textmate-grammars/src/browser/go.ts

```diff
@@ -0,0 +1,75 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
```

Comment:
It is a copy of existing code, so the copyright should be fine.

Reply:
@svenefftinge @marcdumais-work  what about CQs, do we need it when we copy from our repos?

Reply:
Where was this copied-from?

Reply:
probably from theia go extension

Reply:
ok. If so then I think we need a CQ, since that repo is not "Eclipse content" (despite being of the correct license, which should help get the CQ approved, however). 

ref: https://www.eclipse.org/legal/EclipseLegalProcessPoster.pdf

this case: 

![image](https://user-images.githubusercontent.com/25749063/51329574-ce6ea700-1a43-11e9-8957-37f311f8b007.png)

Reply:
Actually I copied it from nowhere.
I created it based on https://github.com/Microsoft/vscode/blob/master/extensions/go/language-configuration.json
If I set `2019 RedHat` will it be ok?

Reply:
> If I set 2019 RedHat will it be ok?

I do not think that's enough. I think it needs to be clear that we have copied code in this file and where it came from, and under which license. Depending on the original license, we may or may not be able to license our own contributions to the file under our own license. 

The CQ is required as described above.